docker-app/qfieldcloud/settings.py

```diff
@@ -49,6 +49,8 @@
 # For example: 'DJANGO_ALLOWED_HOSTS=localhost 127.0.0.1 [::1]'
 ALLOWED_HOSTS = os.environ.get("DJANGO_ALLOWED_HOSTS", "").split(" ")
 
+WEB_HTTPS_PORT = os.environ.get("WEB_HTTPS_PORT")
```

Comment:
```suggestion
WEB_HTTP_PORT = os.environ.get("WEB_HTTP_PORT")
WEB_HTTPS_PORT = os.environ.get("WEB_HTTPS_PORT")
```

Just for the sake of completeness we can have both.